hooks/pre-commit

```diff
@@ -63,8 +63,12 @@ modified_staged_scripts = to_string(subprocess.check_output(
 added_deleted_staged_scripts = to_string(subprocess.check_output(
     "git diff --cached --name-only --diff-filter=AmD scripts", shell=True))
 
+if added_deleted_staged_scripts != "":
+	print("\nThe legacy scripts of Retriever cannot be added to or removed. To add scripts,\
+	use the retriever-recipes repository.")
+	exit(1)
+
```

Comment:
This part is added to prevent adding other scripts prior to a commit. It also tells that the recipes repository would be the correct place to add scripts.